packages/flutter_tools/lib/src/version.dart

```diff
@@ -254,6 +257,8 @@ class FlutterVersion {
     try {
       final String branch = _channel == 'alpha' ? 'alpha' : 'master';
       final DateTime remoteFrameworkCommitDate = DateTime.parse(await FlutterVersion.fetchRemoteFrameworkCommitDate(branch));
+      // fire and forget since nothing subsequent depends on it. Hope there's a flush/sync mechanism.
```

Comment:
This hope seems optimistic...

cc @yjbanov did you mean to not wait for this future?